lars-reimann/api-editor
# refactor: state management

Closes #594.

### Summary of Changes

* New Redux slice for generic UI state
* Persist new UI slice in IndexedDB
* Move UsageCountStore into usage slice
* Move PythonPackage into api slice

### Testing Instructions

1. Open GUI in browser
2. Import API data
3. Import Usage data
4. Import Annotation data
5. Expand everything
6. Set filter to something new (-> tree view should still be filtered properly)
7. Refresh page (-> everything should still be there)
8. Click all buttons
